addons/mail/tests/discuss/test_message_controller.py

```diff
@@ -186,54 +201,6 @@ def test_channel_message_attachments(self):
             ],
             "guest should be allowed to add attachment with token when updating message",
         )
-        # test message update: own attachment ok
-        res5 = self.url_open(
-            url="/mail/message/update_content",
-            data=json.dumps(
-                {
-                    "params": {
-                        "message_id": data2["mail.message"][0]["id"],
-                        "body": "test",
-                        "attachment_ids": [self.attachments[1].id],
-                    },
-                }
-            ),
-            headers={"Content-Type": "application/json"},
-        )
-        self.assertEqual(res5.status_code, 200)
-        data3 = res5.json()["result"]
-        self.assertEqual(
-            data3["ir.attachment"],
-            [
-                {
-                    "checksum": False,
-                    "create_date": fields.Datetime.to_string(self.attachments[0].create_date),
-                    "id": self.attachments[0].id,
-                    "name": "File 1",
-                    "raw_access_token": limited_field_access_token(self.attachments[0], "raw"),
-                    "res_name": "Test channel",
-                    "mimetype": "application/octet-stream",
-                    "thread": {"id": self.channel.id, "model": "discuss.channel"},
-                    "voice": False,
-                    'type': 'binary',
-                    'url': False,
-                },
-                {
-                    "checksum": False,
-                    "create_date": fields.Datetime.to_string(self.attachments[1].create_date),
-                    "id": self.attachments[1].id,
-                    "name": "File 2",
-                    "raw_access_token": limited_field_access_token(self.attachments[1], "raw"),
-                    "res_name": "Test channel",
-                    "mimetype": "application/octet-stream",
-                    "thread": {"id": self.channel.id, "model": "discuss.channel"},
-                    "voice": False,
-                    'type': 'binary',
-                    'url': False,
-                },
-            ],
-            "guest should be allowed to add own attachment without token when updating message",
-        )
```

Comment:
We have another test in this file with sending the wrong token to update your own message, which seems to be the same as not sending the token with the new changes.